code/modules/events/space_ninja.dm

```diff
@@ -35,6 +35,21 @@
 	ninja.key = key
 	ninja.mind.add_antag_datum(/datum/antagonist/ninja)
 	spawned_mobs += ninja
+// SKYRAT EDIT ADDITION BEGIN: Preference Ninjas
+	var/loadme = tgui_input_list(ninja, "Do you wish to load your character slot?", "Load Character?", list("Yes!", "No, I want to be random!"), default = "No, I want to be random!", timeout = 60 SECONDS)
+	var/codename
+	switch(loadme)
```

Comment:
what is the point of this switch if it's literally just an if/else

Reply:
in my head this saves a whole microsecond of compute time but idk lol